tests/unit/tickstore/test_tickstore.py

```diff
@@ -43,3 +45,48 @@ def test_strify_tickstore():
     self = create_autospec(TickStore)
     self._arctic_lib = sentinel.library
     assert 'sentinel.library' in TickStore.__str__(self)
+
+
+def test_tickstore_to_bucket_no_image():
+    symbol = 'SYM'
+    data = [{'index': dt(2014, 1, 1, 0, 1, tzinfo=mktz()), 'A': 124, 'D': 0},
+            {'index': dt(2014, 1, 1, 0, 2, tzinfo=mktz()), 'A': 125, 'B': 27.2}]
+    bucket, final_image = TickStore._to_bucket(data, symbol, None)
+    assert bucket[COUNT] == 2
+    assert bucket[END] == dt(2014, 1, 1, 0, 2, tzinfo=mktz())
+    assert bucket[SYMBOL] == symbol
+    assert bucket[START] == dt(2014, 1, 1, 0, 1, tzinfo=mktz())
+    assert 'A' in bucket[COLUMNS]
+    assert IMAGE_DOC not in bucket
+    assert not final_image
+
+
+def test_tickstore_to_bucket_with_image():
+    symbol = 'SYM'
+    initial_image = {'index': dt(2014, 1, 1, 0, 0, tzinfo=mktz()), 'A': 123, 'B': 54.4, 'C': 'DESC'}
+    data = [{'index': dt(2014, 1, 1, 0, 1, tzinfo=mktz()), 'A': 124, 'D': 0},
+            {'index': dt(2014, 1, 1, 0, 2, tzinfo=mktz()), 'A': 125, 'B': 27.2}]
+    bucket, final_image = TickStore._to_bucket(data, symbol, initial_image)
+    assert bucket[COUNT] == 2
+    assert bucket[END] == dt(2014, 1, 1, 0, 2, tzinfo=mktz())
+    assert bucket[SYMBOL] == symbol
+    assert bucket[START] == dt(2014, 1, 1, 0, 0, tzinfo=mktz())
+    assert final_image == {'index': dt(2014, 1, 1, 0, 2, tzinfo=mktz()), 'A': 125, 'B': 27.2, 'C': 'DESC', 'D': 0}
+
+
+def test_tickstore_pandas_to_bucket_image():
+    symbol = 'SYM'
+    initial_image = {'index': dt(2014, 1, 1, 0, 0, tzinfo=mktz()), 'A': 123, 'B': 54.4, 'C': 'DESC'}
+    data = [{'A': 120, 'D': 1}, {'A': 122, 'B': 2.0}, {'A': 3, 'B': 3.0, 'D': 1}]
+    data = pd.DataFrame(data, index=[dt(2014, 1, 2, 0, 0, tzinfo=mktz()), dt(2014, 1, 3, 0, 0, tzinfo=mktz()),
+                                     dt(2014, 1, 4, 0, 0, tzinfo=mktz())])
+    bucket, final_image = TickStore._pandas_to_bucket(data, symbol, initial_image)
```

Comment:
I think we should be asserting the contents of the bucket here. It would be useful to lock down the contents of the IMAGE subdocument.